client/keys/utils.go

```diff
@@ -28,34 +28,35 @@ const (
 type bechKeyOutFn func(keyInfo keys.Info) (keys.KeyOutput, error)
 
 // NewKeyBaseFromHomeFlag initializes a Keybase based on the configuration.
-func NewKeyBaseFromHomeFlag() (keys.Keybase, error) {
+func NewKeyBaseFromHomeFlag(opts ...keys.KeybaseOption) (keys.Keybase, error) {
```

Comment:
godoc string needs update

Reply:
Will update all of these now, not really sure exactly how you would like this worded but I'll give it a shot ! :)

Reply:
This function is exported, and is an example of API breaking change @austinabell